frontend/src/__tests__/cypress/cypress/pages/enabled.ts

```diff
@@ -0,0 +1,52 @@
+class EnabledPage {
+  visit() {
+    cy.visitWithLogin('/');
+    this.wait();
+  }
+
+  private wait() {
+    cy.findByTestId('enabled-application').should('be.visible');
+    cy.testA11y();
+  }
+
+  findPageTitle() {
+    return cy.findByTestId('app-page-title');
+  }
+
+  shouldHavePageDescription() {
+    return cy
+      .findByTestId('app-page-description')
+      .contains(
+        'Launch your enabled applications, view documentation, or get started with quick start instructions and tasks.',
+      );
+  }
+
+  findEnabledCard() {
+    return cy.findByTestId('jupyter');
+  }
+
+  findBrandImage() {
+    return cy.findByTestId('Jupyter-image');
+  }
+
+  findCardTitle() {
+    return cy.findByTestId('Jupyter-title');
+  }
+
+  findCertifiedImage() {
+    return cy.findByTestId('Open Data Hub');
+  }
+
+  findTooltipInfo() {
+    return cy.findByTestId('tool-tip');
+  }
+
+  findPartnerBadge() {
+    return cy.findByTestId('Red Hat managed');
+  }
+
+  findPartnerBadgeDescription() {
+    return cy.findByTestId('badge-description');
+  }
```

Comment:
If multiple cards exist on the page, every one of these selectors will fail. You must create a scoped page object for the card.

Reply:
@christianvogt can you give me an example?

Reply:
If a test ID appears more than once on the page, then it must be scoped. Since a card can appear multiple times within a gallery, the selectors for the card should be scoped. See examples of how we do it for modals and table rows.

First create a page object for the card, then add the necessary `find` functions to retrieve the `title` etc...

For example, asserting the title should look something like this:
```
const jupyterCard = enabledPage.getCard('card jupyter');
jupyterCard.findTitle().should('have.text', 'Jupyter');
jupyterCard.find...
```